### ISSUE TITLE
`Lint/EmptyConditionalBody` does not allow commented empty body in 2nd `elsif`

### ISSUE THREAD

**Issue by r7kamura** [rubocop/rubocop#11820]
## Expected behavior

In `Lint/EmptyConditionalBody`, when `AllowComments: true`, no offense should be detected from the following code:

```ruby
if condition1
  do_something1
elsif condition2
  do_something2
elsif condition3
  # noop
end
```

## Actual behavior

`Lint/EmptyConditionalBody` detects an offense like this:

```
elsif condition2
  do_something2
elsif condition3
+^^^^^^^^^^^^^^^^ Avoid `elsif` branches without a body.
  # noop
end
```

## Steps to reproduce the problem

This behavior can be reproduced by applying the following patch:

```patch
diff --git a/spec/rubocop/cop/lint/empty_conditional_body_spec.rb b/spec/rubocop/cop/lint/empty_conditional_body_spec.rb
index edd5cda13..8ceea38f8 100644
--- a/spec/rubocop/cop/lint/empty_conditional_body_spec.rb
+++ b/spec/rubocop/cop/lint/empty_conditional_body_spec.rb
@@ -75,6 +75,18 @@ RSpec.describe RuboCop::Cop::Lint::EmptyConditionalBody, :config do
     RUBY
   end
 
+  it 'does not register an offense for missing 2nd `elsif` body with a comment' do
+    expect_no_offenses(<<~'RUBY')
+      if condition1
+        do_something1
+      elsif condition2
+        do_something2
+      elsif condition3
+        # noop
+      end
+    RUBY
+  end
+
   it 'registers an offense for missing `elsif` body' do
     expect_offense(<<~RUBY)
       if condition
```

## RuboCop version

````
$ bundle exec rubocop -V
1.50.2 (using Parser 3.2.2.1, rubocop-ast 1.28.0, running on ruby 3.1.3) [x86_64-linux]
  - rubocop-performance 1.17.1
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.20.0
````
